cfg/config.yaml

```diff
@@ -28,6 +28,7 @@ master:
       - "apiserver"
       - "openshift start master api"
       - "hypershift openshift-kube-apiserver"
+      - "containerd"
```

Comment:
IMHO, we should refrain from adding `containerd` to every configuration in this file. For this specific case, the configuration should be specified in each benchmark's dedicated configuration file, such as `cfg/k3s-cis-1.23/config.yaml`.